strings/en/prefix.json

```diff
@@ -16,8 +16,8 @@
 	"caution": "Caution",
 	"warning": "Setting your prefix to a language you don't translate will result in a warning. Do you want to change your nickname anyway?",
 	"react": "React with all flags you want to add to your prefix in order.",
-	"reactTimer": "You have %%cooldown%% seconds. Hit ❎ to cancel.",
-	"reactTimer2": "You have less than %%cooldown%% seconds left. Hit ✅ to confirm or ❎ to cancel.",
+	"reactTimer": "You have {cooldown, plural, one {# second} other {# seconds}}. Hit ❎ to cancel.",
+	"reactTimer2": "You have less than {cooldown, plural, one {# second} other {# seconds}} left. Hit ✅ to confirm or ❎ to cancel.",
```

Comment:
is this needed? These are fixed values that are either 30 or 60 seconds

Reply:
Some languages may have different words for 30 and 60

Reply:
What I'm saying is this is likely always the same value, I can double check but I'm pretty sure it is